sdk/anomalydetector/ai-anomaly-detector-rest/samples-dev/detectEntireSeriesSample.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createAnomalyDetectorRestClient, {
+  DetectEntireSeriesParameters,
+} from "@azure-rest/ai-anomaly-detector";
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to This operation generates a model with an entire series, each point is detected with the same model. With this method, points before and after a certain point are used to determine whether it is an anomaly. The entire detection can give user an overall status of the time series.
+ *
+ * @summary This operation generates a model with an entire series, each point is detected with the same model. With this method, points before and after a certain point are used to determine whether it is an anomaly. The entire detection can give user an overall status of the time series.
+ * x-ms-original-file: specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/examples/EntireDetect.json
+ */
+async function findAnomaliesForTheEntireSeriesInBatchExample() {
+  const Endpoint = "{Endpoint}";
+  const ApiVersion = "v1.1";
+  const credential = new AzureKeyCredential("{Your API key}");
+  const client = createAnomalyDetectorRestClient(Endpoint, ApiVersion, credential);
+  const options: DetectEntireSeriesParameters = {
+    body: {
+      granularity: "monthly",
+      imputeMode: "auto",
+      maxAnomalyRatio: 0.25,
+      sensitivity: 95,
+      series: [
+        { timestamp: new Date("1972-01-01T00:00:00Z"), value: 826 },
+        { timestamp: new Date("1972-02-01T00:00:00Z"), value: 799 },
+        { timestamp: new Date("1972-03-01T00:00:00Z"), value: 890 },
+        { timestamp: new Date("1972-04-01T00:00:00Z"), value: 900 },
+        { timestamp: new Date("1972-05-01T00:00:00Z"), value: 961 },
+        { timestamp: new Date("1972-06-01T00:00:00Z"), value: 935 },
+        { timestamp: new Date("1972-07-01T00:00:00Z"), value: 894 },
+        { timestamp: new Date("1972-08-01T00:00:00Z"), value: 855 },
```

Comment:
Remove the `new Date()` wrapper code, the behavior is the same either way.

Reply:
Done